Automattic/wp-calypso
# Show reviews modal on theme details page

Closes https://github.com/Automattic/dotcom-forge/issues/5268

## Proposed Changes

* Show the reviews modal when clicking the reviews link
* Only show the add review button if the theme doesn't have 3 reviews yet

## Testing Instructions



* Go to a theme that doesn't have 3 reviews, eg. `/theme/organic-stax`
* Check that the stats aren't shown
![CleanShot 2024-01-23 at 11 38 47@2x](https://github.com/Automattic/wp-calypso/assets/11555574/189f9b56-1c2a-404e-8fdd-90222b715a70)
* Go to a theme that has 3 or more reviews, eg. `/theme/spiel`
* Check that the stats are shown
![CleanShot 2024-01-23 at 11 37 50@2x](https://github.com/Automattic/wp-calypso/assets/11555574/45064307-8d1c-4764-95d9-7cd92f84b227)
* Click the reviews link
* Check that the reviews modal is opened and shows the reviews correctly
  ![CleanShot 2024-01-23 at 15 36 19@2x](https://github.com/Automattic/wp-calypso/assets/11555574/06fc102c-1ca7-44d0-8e45-2a8b3af3d992)


## Pre-merge Checklist
